Inspicio/Controllers/AccountController.cs

```diff
@@ -122,6 +122,12 @@ public IActionResult Register(string returnUrl = null)
             if (ModelState.IsValid)
             {
                 var user = new ApplicationUser { UserName = model.Email, Email = model.Email };
```

Comment:
Decide on naming conventions - either upper case for all the first letters of the word, or camel case etc. At the moment there is some variation between the classes.